streampark-console/streampark-console-service/src/main/assembly/script/schema/mysql-schema.sql

```diff
@@ -556,4 +556,97 @@ create table if not exists `t_resource` (
 `modify_time` datetime not null default current_timestamp on update current_timestamp comment 'modify time'
 );
 
+
+-- ----------------------------
+-- table structure for t_spark_env
+-- ----------------------------
+drop table if exists `t_spark_env`;
+create table `t_spark_env` (
+                               `id` bigint not null auto_increment comment 'id',
+                               `spark_name` varchar(128) collate utf8mb4_general_ci not null comment 'spark instance name',
+                               `spark_home` varchar(255) collate utf8mb4_general_ci not null comment 'spark home path',
+                               `version` varchar(64) collate utf8mb4_general_ci not null comment 'spark version',
+                               `scala_version` varchar(64) collate utf8mb4_general_ci not null comment 'scala version of spark',
+                               `spark_conf` text collate utf8mb4_general_ci not null comment 'spark-conf',
+                               `is_default` tinyint not null default 0 comment 'whether default version or not',
+                               `description` varchar(255) collate utf8mb4_general_ci default null comment 'description',
+                               `create_time` datetime not null default current_timestamp comment 'create time',
```

Comment:
`default current_timestamp` has syntax compatibility issue, change to : `default null`